### DISCUSSION TITLE
Sporadic very high readings

### DISCUSSION THREAD

**Discussion by beta-j** [klaasnicolaas/home-assistant-glow#93] [Q&A] [answered]
I've been using Glow integrated with my Home Assistant setup for just over a month now and I think it's great.  

My only problem is that for some reason I get extremely high readings every so often and always of 40,449W.  These seem to happen at random and of course, they are incorrect readings (My house is protected by a 40A fuse on 230V AC).  It's not a big problem, but it does entirely mess up the scale of the history graphs in home assistant (see screenshot).

Has anyone else experienced this? Can anyone figure out where the specific 40449 figure is coming from maybe?

![spikes](https://user-images.githubusercontent.com/60655500/143557583-79509045-b392-4b7f-b65b-a6075ff31e8a.png)

**Comment by lemppari** [comment, marked as answer]
Check out Issue 57 (https://github.com/klaasnicolaas/home-assistant-glow/issues/57), there is some discussion over similar problems.

**Reply by beta-j** [reply]
Thanks for pointing me in the right direction.  I've enabled the internal filter and I'll look into the possibility of RF shielding for the cables connecting the light detector to the esp board and see how it goes :)
